src/librustc_target/spec/aarch64_uwp_windows_msvc.rs

```diff
@@ -0,0 +1,29 @@
+use crate::spec::{LinkerFlavor, Target, TargetResult, PanicStrategy};
+use std::env;
+
+pub fn target() -> TargetResult {
+    let mut base = super::windows_uwp_msvc_base::opts();
+    base.max_atomic_width = Some(64);
+    base.has_elf_tls = true;
+
+    // FIXME: this shouldn't be panic=abort, it should be panic=unwind
```

Comment:
What's the background on this FIXME that causes it to exist? Does that belong in a comment or an issue?

Reply:
The AArch64 Windows backend didn't implement SEH unwinding when the aarch64-pc-windows-msvc target was first added, so I suspect this is the same as that. I haven't checked the LLVM backend recently to see if this works.